packages/amplify-category-geo/src/service-utils/populateParams.ts

```diff
@@ -0,0 +1,40 @@
+export type FeatureCollection = {
```

Comment:
reviewed in https://github.com/aws-amplify/amplify-cli/pull/8822/files#diff-91c46f444843f8e496039f2da5af36ae64adc134aee76009b54097142411fbfdR1